configure

```diff
@@ -304,6 +304,31 @@ else
 	fail "$0: cannot find libtermkey"
 fi
 
+cat > "$tmpc" <<EOF
+#include <sregex.h>
+
+int main(int argc, char *argv[]) {
+    sre_pool_t *p = sre_create_pool(1024);
+    sre_destroy_pool(p);
+    return 0;
+}
+EOF
+
+CONFIG_SREGEX=0
+
+printf "checking for sregex... "
+
+CFLAGS_SREGEX="-I${PREFIX}/include/sregex"
+LDFLAGS_SREGEX="-L${PREFIX}/lib -Wl,-rpath=${PREFIX}/lib -lsregex"
```

Comment:
So far we avoided specifying `-rpath` and instead advised users to set `$LD_LIBRARY_PATH` as required.